src/platform/silabs/wifi/WifiInterfaceAbstraction.h

```diff
@@ -118,9 +118,9 @@ typedef enum
 
 typedef struct
 {
-    char ssid[WFX_MAX_SSID_LENGTH + 1];
+    char ssid[WFX_MAX_SSID_LENGTH + 1] = {0};
```

Comment:
Why all the elements of the ssid array is initialized to zero? observed any issue?

Reply:
If is not initialized some times it is having garbage value and condition is passing even though DUT is not provisioned.